heartpy/heartpy.py

```diff
@@ -294,6 +295,23 @@ def process(hrdata, sample_rate, windowsize=0.75, report_time=False,
                                          desired_sample_rate=high_precision_fs, working_data=working_data)
 
     working_data = calc_rr(working_data['peaklist'], sample_rate, working_data=working_data)
+    
+    # Obtain Mode
+    mode_v = mode(working_data['RR_list'])
+    measures['Mo'] = mode_v
+
+    # Obtain VR (variational range)
+    vr = np.max(working_data['RR_list']) - np.min(working_data['RR_list'])
+    measures['VR'] = vr
+
+    # Obtain AMo (mode amplitude %)
+    num_mode = np.count_nonzero(working_data['RR_list'] == mode_v)
+    amo = (num_mode/len(working_data['RR_list']))*100
+    measures['AMo'] = amo
+
+    # Calculate Baevsky Stress Index
+    si = (amo/(2*vr*mode_v)) * 100
+    measures['SI'] = si
```

Comment:
I would love to see the square root of the SI metric here as well. as this is what Kubios also applies to transform the tailed distribution to a normal distribution (see https://www.kubios.com/hrv-analysis-methods/)